lib/git.zsh

```diff
@@ -36,7 +36,10 @@ git_remote_status() {
         ahead=$(command git rev-list ${hook_com[branch]}@{upstream}..HEAD 2>/dev/null | wc -l)
         behind=$(command git rev-list HEAD..${hook_com[branch]}@{upstream} 2>/dev/null | wc -l)
 
-        if [ $ahead -gt 0 ] && [ $behind -eq 0 ]
+        if [ $ahead -eq 0 ] && [ $behind -eq 0 ]
+        then
+          echo "$ZSH_THEME_GIT_PROMPT_EQUAL_REMOTE"
```

Comment:
I tried to reproduce the problem, without success. Looks good to me:
<img width="590" alt="screen shot 2015-09-22 at 9 51 30 pm" src="https://cloud.githubusercontent.com/assets/530992/10029773/241d7370-6174-11e5-826c-3b4d42f1b793.png">